app/server-utils/stampy.ts

```diff
@@ -1,4 +1,5 @@
 import {withCache} from '~/server-utils/kv-cache'
+import {urlToIframe} from './url-to-iframe'
```

Comment:
could you change this to `'~/server-utils/url-to-iframe'`, for consistency?

Reply:
Ah yes, certainly, thanks for pointing that out.

Reply:
Fixed in https://github.com/StampyAI/stampy-ui/pull/148/commits/dd14d29b973ee23221cbe391c9e1c1535d4653c1